lib/src/layer/polygon_layer/painter.dart

```diff
@@ -256,10 +302,13 @@ class _PolygonPainter extends CustomPainter {
     path.addOval(Rect.fromCircle(center: offsets.last, radius: radius));
   }
 
-  void _addLineToPath(ui.Path path, List<Offset> offsets) {
+  void _addLineToPath(Path path, List<Offset> offsets) {
     path.addPolygon(offsets, true);
   }
 
+  // TODO: Fix bug where wrapping layer in some widgets (eg. opacity) causes the
+  // features to not move unless this is `true`, but `true` significantly impacts
+  // performance
```

Comment:
Is this only the case for drawVertices? 

If so we should 
- add this to the documentation of `performantRendering`
- reconsider if `performantRendering` should be true by default of rather false (aka use this but be aware of the potential consequences).

Reply:
This is unrelated to `drawVertices`, and is a bug I noticed/was reported whilst developing this. Also, "significant" was probably OTT, but there is still a small performance difference.